bitnami/postgresql-ha/values.yaml

```diff
@@ -310,153 +276,166 @@ postgresql:
+  ## PostgreSQL container's startup probe
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  ## @param postgresql.startupProbe.enabled Enable startupProbe
+  ## @param postgresql.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param postgresql.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param postgresql.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param postgresql.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param postgresql.startupProbe.successThreshold Success threshold for startupProbe
+  ##
   startupProbe:
     enabled: false
     initialDelaySeconds: 5
     periodSeconds: 10
     timeoutSeconds: 5
     successThreshold: 1
     failureThreshold: 10
-
   ## Pod disruption budget configuration
   ##
   pdb:
-    ## Specifies whether a Pod disruption budget should be created
+    ## @param postgresql.pdb.create Specifies whether to create a Pod disruption budget for PostgreSQL with Repmgr
     ##
     create: false
+    ## @param postgresql.pdb.minAvailable Minimum number / percentage of pods that should remain scheduled
+    ##
     minAvailable: 1
-    # maxUnavailable: 1
-
+    ## @param postgresql.pdb.maxUnavailable Maximum number / percentage of pods that may be made unavailable
+    ##
+    maxUnavailable:
   ## PostgreSQL configuration parameters
+  ## @param postgresql.username PostgreSQL username
+  ## @param postgresql.password PostgreSQL password
+  ## @param postgresql.database PostgreSQL database
   ##
   username: postgres
-  # password:
-  # database:
-
-  ## PostgreSQL password using existing secret
-  # existingSecret: secret
-
-  ## PostgreSQL admin password (used when `postgresql.username` is not `postgres`)
+  password:
+  database:
+  ## @param postgresql.existingSecret PostgreSQL password using existing secret
+  ##
+  existingSecret:
+  ## @param postgresql.postgresPassword PostgreSQL password for the `postgres` user when `username` is not `postgres`
   ## ref: https://github.com/bitnami/bitnami-docker-postgresql/blob/master/README.md#creating-a-database-user-on-first-run (see note!)
-  # postgresPassword:
-
-  ## Mount PostgreSQL secret as a file instead of passing environment variable
-  # usePasswordFile: false
-
-  ## Store Repmgr username and password into a passfile with format "*:*:*:username:password" and use it to configure Repmgr instead of using password (Requires Postgresql 10+, otherwise ignored)
-  # ref: https://repmgr.org/docs/current/configuration-password-management.html
-  #      https://www.postgresql.org/docs/current/libpq-pgpass.html
-  # repmgrUsePassfile: false
-
-  ## Store Repmgr in a custom path
-  # repmgrPassfilePath:
-
-  ## Upgrade repmgr extension in the database
+  ##
+  postgresPassword:
+  ## @param postgresql.usePasswordFile Set to `true` to mount PostgreSQL secret as a file instead of passing environment variable
+  ##
+  usePasswordFile:
```

Comment:
Attention with this, this key was commented and right now no. Maybe this is going to change the default behavior of the chart. Did you test this change?

Reply:
Verified that the manifests generated by the default values are not affected by diffing:
```bash
git checkout master values.yaml && helm install postgresql-ha . -f values.yaml --dry-run --debug > pre.yaml
```
```bash
git checkout HEAD values.yaml && helm install postgresql-ha . -f values.yaml --dry-run --debug > post.yaml
```